ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -25,104 +25,102 @@
  * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
  * OTHER DEALINGS IN THE SOFTWARE.
  */
+
 #include <wiredtiger.h>
```

Comment:
Did you check that all the includes are necessary?

Reply:
Good point, we can remove the previous filesystem headers.